hls4ml/templates/vivado/build_prj.tcl

```diff
@@ -91,10 +189,27 @@ if {$opt(cosim)} {
   # TODO: This is a workaround (Xilinx defines __RTL_SIMULATION__ only for SystemC testbenches).
   add_files -tb myproject_test.cpp -cflags "-std=c++0x -DRTL_SIM"
   set time_start [clock clicks -milliseconds]
-  cosim_design -trace_level all
+
+  cosim_design -trace_level all -setup
+
+  if {$fifo_opt} {
```

Comment:
Can we add this as an option in the `set opt`, and to an argument in `build` rather than adding it to this `project.tcl`? It is really a build option rather than a project setting